Retitle the Fleet data retention tutorials with task-based titles (#7853)
elastic/docs-content: reference/fleet/integrations-assets-best-practices.md

Additionally, cloning index templates to add or inject additional component templates cannot be tested by Elastic, so we cannot guarantee that the template will work in future releases.

> If you want to change the ILM Policy, the number of shards, or other settings for the data streams of one or more integrations, but the changes do not need to be specific to a given namespace, it’s highly  recommended to use the `package@custom` component templates, as described in [Apply an ILM policy to all data streams across all namespaces](/reference/fleet/data-streams-scenario1.md) and [Apply an ILM policy to specific data streams across all namespaces](/reference/fleet/data-streams-scenario2.md), so as to avoid the problems mentioned above.

Author: ```suggestion
If you want to change the ILM Policy, the number of shards, or other settings for the data streams of one or more integrations, but the changes do not need to be specific to a given namespace, it’s highly  recommended to use the `package@custom` component templates, as described in [Apply an ILM policy to all data streams across all namespaces](/reference/fleet/data-streams-scenario1.md) and [Apply an ILM policy to specific data streams across all namespaces](/reference/fleet/data-streams-scenario2.md), so as to avoid the problems described earlier on this page.
```

> If you want to change the ILM Policy, the number of shards, or other settings for the data streams of one or more integrations, but the changes do not need to be specific to a given namespace, it’s highly  recommended to use the `package@custom` component templates, as described in [Apply an ILM policy to all data streams across all namespaces](/reference/fleet/data-streams-scenario1.md) and [Apply an ILM policy to specific data streams across all namespaces](/reference/fleet/data-streams-scenario2.md), so as to avoid the problems described earlier on this page.

Reviewer: ```suggestion
If you want to change the ILM Policy, the number of shards, or other settings for the data streams of one or more integrations, but the changes do not need to be specific to a given namespace, it’s highly recommended to use the `package@custom` component templates, as described in [Apply an ILM policy to all data streams across all namespaces](/reference/fleet/data-streams-scenario1.md) and [Apply an ILM policy to specific data streams across all namespaces](/reference/fleet/data-streams-scenario2.md), so as to avoid the problems described earlier on this page.
```
